vnext/Microsoft.ReactNative/Views/Image/ReactImage.cpp

```diff
@@ -342,7 +342,20 @@ winrt::fire_and_forget ReactImage::SetBackground(bool fireLoadEndEvent) {
         }
 
         if (fromStream) {
-          co_await svgImageSource.SetSourceAsync(memoryStream);
+          try {
+            co_await svgImageSource.SetSourceAsync(memoryStream);
+          } catch (const winrt::hresult_error&) {
+            /*
+                winrt::hresult_canceled
```

Comment:
Can we filter to this specific hresult rather than eat all failures to SetSourceAsync?

Reply:
that's what I thought about doing at first (just cancellation), but then I saw there's another comment related to "WIC can throw if the image is 'bad'" for some definition of bad so I figured better safe than sorry. In general it feels better to not expose the app to native crashes. Thoughts?

Reply:
Is there another appropriate place to report/log the error? Events of some sort? For example there's some sort of ImageFailed doing... debug stuff. And in GetImageStreamAsync it handles the error with some macro.
@marlenecota do you know what's appropriate for Image to report?

Reply:
m_onLoadEndEvent is an event that would get fired back to the JS to report the load state / failure of the image.

Reply:
the js failure would triggered if we got that far, however the code as it exists before this change crashes before we get a chance to call the image load failure event handler